doc/source/upgrading_guide.rst

```diff
@@ -322,13 +330,13 @@ After you unpack the bundle you should see the following packages: :bash:`ls *.r
       percona-server-test-80-8.0.13-3.el7.x86_64.rpm
       percona-server-tokudb-80-8.0.13-3.el7.x86_64.rpm
 
-Now you can install |Percona Server| 8.0 by running:
+Now, you can install |Percona Server| 8.0 by running:
 
 .. code-block:: bash
 
-   rpm -ivh percona-Server-server-8.0.13-3.el7.x86_64.rpm \
-   percona-perver-client-8.0.13-3.el7.x86_64.rpm \
-   percona-server-shared-8.0.13-3.el7.x86_64.rpm
+   rpm -ivh percona-server-server_8.0.13-3.el7.x86_64.rpm \
+   percona-perver-client_8.0.13-3.el7.x86_64.rpm \
```

Comment:
* `perver`
* packages have mix of `_` and `-` in their names

Here is the list of package names from CentOS 6:
```
[root@centos6 ~]# rpm -qa | grep percona-server
percona-server-devel-8.0.13-3.1.el6.x86_64
percona-server-rocksdb-8.0.13-3.1.el6.x86_64
percona-server-shared-8.0.13-3.1.el6.x86_64
percona-server-server-8.0.13-3.1.el6.x86_64
percona-server-test-8.0.13-3.1.el6.x86_64
percona-server-tokudb-8.0.13-3.1.el6.x86_64
percona-server-debuginfo-8.0.13-3.1.el6.x86_64
percona-server-shared-compat-8.0.13-3.1.el6.x86_64
percona-server-client-8.0.13-3.1.el6.x86_64
```